packages/ui/lib/toast/transaction-toast.tsx

```diff
@@ -49,6 +50,14 @@ export class TransactionToast {
 
   constructor(private transactionClassification: TransactionClassification) {}
 
+  private get label(): string {
+    return TRANSACTION_LABEL_BY_CLASSIFICATION[this.transactionClassification];
+  }
+
+  private get shortenedTxHash(): string {
+    return this._txHash ? shorten(this._txHash, 8) : '';
+  }
+
```

Comment:
Was this an automatic change from your IDE? I feel like it's generally the norm to put private methods last.